src/index.js

```diff
@@ -1,6 +1,9 @@
 //! Default Compute@Edge template program.
 
 /// <reference types="@fastly/js-compute" />
+
+// import { CacheOverride } from "fastly:cache-override";
+// import { Logger } from "fastly:logger";
 import welcomePage from "./welcome-to-compute@edge.html";
```

Comment:
Instead of relying on webpack here, we could use `import { includeBytes } from "fastly:experimental"`

With this change, there is actually no need to use webpack in this project at all and we can rely solely on js-comtpute-runtime's support for `import` and `export`

Reply:
Yeah thanks for the input. For this PR we'll do it this way but will make one w/o webpack soon.